services/FeefoReviewService.php

```diff
@@ -29,19 +27,22 @@ public function feefoMerchantId() {
 
   private function getProducts() {
     $cacheKey = 'feefo_'.$this->feefoMerchantId();
-    $products = craft()->cache->get($cacheKey);
-    if ($products === false) {
-      $products = $this->getProductReviews();
-      if (!is_null($products))
-        craft()->cache->set($cacheKey, $products, 24*3600 /*24 hours*/);
+    $products = $this->getProductReviews();
```

Comment:
This code does not cache like it did before. The cache is completely bypassed in your version. Please have a read through the previous version of this function and let me know if you need any assistance understanding why this is.